setup.py

```diff
@@ -218,7 +218,8 @@ def run(self):
     ':python_version == "2.7" or python_version == "3.3"': ['pathlib2'],
     ':sys_platform != "win32"': ['pexpect'],
     ':sys_platform == "darwin"': ['appnope'],
-    ':sys_platform == "win32"': ['colorama', 'win_unicode_console>=0.5'],
+    ':sys_platform == "win32"': ['colorama'],
+    ':sys_platform == "win32" and python_version < "3.6"': ['win_unicode_console>=0.5'],
```

Comment:
We already have similar ones above, I suspect we already require something similar.